src/payment-flow-handlers/handlers/paypal-flow-handler.ts

```diff
@@ -302,45 +280,4 @@ export class PayPalFlowHandler
       console.error('error rendering paypal upsell button');
     }
   }
-
-  private buildDonationRequest(params: {
```

Comment:
where does this happen now?

Reply:
This was actually not being used, but I forgot to remove it when I refactored this a little while back. The donation request building now happens in [`BraintreeManager.submitDonation()`](https://github.com/internetarchive/iaux-donation-form/pull/1/files#diff-5fcc84b3c7d2a9a0f859f85fac8447eeR21). 

Each of the flow handlers pass in the individual pieces of information that are needed to submit the request to that method and the `BraintreeManager` does the rest. Prior to the refactor, each of the flow handlers was building this independently like this since I couldn't figure out where the consistencies were between them.